lib/msal-browser/src/config/Configuration.ts

```diff
@@ -85,6 +85,10 @@ export type CacheOptions = {
      * If set, MSAL sets the "Secure" flag on cookies so they can only be sent over HTTPS. By default this flag is set to false.
      */
     secureCookies?: boolean;
+    /**
+     * If set, MSAL will attempt to migrate cache entries from older versions on initialization. By default this flag is set to true if cacheLocation is localStorage, otherwise false.
+     */
+    cacheMigrationEnabled?: boolean;
```

Comment:
Can we update the configuration doc as well?

Reply:
Done